src/query-builder/delete-query-builder.ts

```diff
@@ -498,12 +497,22 @@ export class DeleteQueryBuilder<DB, TB extends keyof DB, O>
     })
   }
 
-  returningAll(): DeleteQueryBuilder<DB, TB, Selectable<DB[TB]>> {
+  returningAll<T extends TB>(
```

Comment:
One last thing..

Let's add ts doc to this one with some examples (e.g. single-table no args, multi-table 1 arg). You can lift them straight from the unit tests.

Reply:
On it. :+1: